### PR TITLE
config: Add 1LC example configuration and accompanying flashing script

### PR DESCRIPTION
**Pull request by Wulfsta**
This adds an example config and a flashing script for the Duet3D Toolboard 1LC. 

@eamaclean please have a look at this when you get a chance, would like your thoughts.

### CODE REVIEW

File: config/generic-duet3-1lc.cfg [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# This file contains common pin mappings for the Duet3 1LC. To use
+# this config, the firmware should be compiled for the SAMC21G18 with:
+# Bootloader offset of "No Bootloader"
+# Clock Reference of "25 Mhz crystal"
```
Comment (eamaclean) [inline config/generic-duet3-1lc.cfg:4]: Should mention the 12MHz crystal on 0.6 and 1.0 boards

File: config/generic-duet3-1lc.cfg [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# Communication interface of "CAN bus (on PA25/PA24)"
+
+# To flash the board use a debugger, or use a raspberry pi and follow
+# the instructions at klipper/scripts/flash-atsamc21.sh. You may
+# supply power to the 1LC by connecting the 5v rail on the Pi to the
```
Comment (eamaclean) [inline config/generic-duet3-1lc.cfg:9]: I would suggest using the 3.3V supply out of an abundance of caution - Pi GPIO is fragile.

File: config/generic-duet3-1lc.cfg (hunk 1 above)
Comment (Wulfsta) [reply config/generic-duet3-1lc.cfg:4]: Good point.

File: config/generic-duet3-1lc.cfg (hunk 2 above)
Comment (Wulfsta) [reply config/generic-duet3-1lc.cfg:9]: I haven’t tried this, honestly not sure if it will work…

Comment (eamaclean) [reply config/generic-duet3-1lc.cfg:9]: It works - the CAN transceiver won't operate at 3.3V but the MCU runs just fine.

**Comment by KevinOConnor** [comment]
Thanks.  The convention is to add toolhead board configs using the "sample" prefix and not the "generic" prefix (as all "generic" configs would need to be added to the regression test cases, which a toolhead board can pass on its own) - https://www.klipper3d.org/Example_Configs.html

I'm not sure it makes sense to add a script for flashing this toolhead board, as it depends on installing/building openocd which I suspect most users wont be prepared to do.  Consider adding pertinent information to docs/Bootloaders.md instead (which already has information on installing and using openocd).

Cheers,
-Kevin

**Comment by KevinOConnor** [comment]
Thanks.

-Kevin

**Pull request merged**